week-2/stretch/till.js

```diff
@@ -22,11 +23,29 @@ const till = {
   "20p": 10,
 };
 const totalAmount = totalTill(till);
-
+console.log(totalAmount);
 // a) What is the target output when totalTill is called with the till object
-
+//  10,
+// "30,
+// "200,
+// "200
+// 4.4
 // b) Why do we need to use Object.entries inside the for...of loop in this function?
-
+// to convert the object in to an array pairs so we can calculate their multiply.
 // c) What does coin * quantity evaluate to inside the for...of loop?
-
+// it multiplies each pairs inside the loop 
 // d) Write a test for this function to check it works and then fix the implementation of totalTill
+test('calculate total amount in pound', (){
```

Comment:
Be careful with the syntax for tests. In this case, the line should end `() => {`